packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -85,7 +86,14 @@ export const networks: { [networkId: number]: string } = Object.freeze({
   77: 'sokol',
   100: 'xdai',
 });
-export const networkIds = (Object.freeze(invert({ ...networks })) as unknown) as { [networkName: string]: number };
+
+// invert the networks object, so { '1': 'mainnet', ... } becomes { mainnet: '1', ... }
+// then map over the values, so that { mainnet: '1', ... } has its values casted as numbers: { mainnet: 1, ... }
+export const networkIds = (Object.freeze(
+  mapValues(invert({ ...networks }), (networkIdString: string) => +networkIdString)
```

Comment:
I think `Number(networkIdString)` or parseInt(networkIdString)` is a little more evident than `+networkIdString`

Reply:
Makes sense! Used `Number`.